SNIPS/snip-template copy.md

```diff
@@ -0,0 +1,49 @@
+---
+snip: $SNIP_ID
+title: SNIP Purpose and Guidelines
+author: $SNIP_AUTHOR <$SNIP_AUTHOR_EMAIL>
+status: Living
+type: Meta
+created: $SNIP_DATE
+---
+
+## Simple Summary
+
+This snip will deal with the emergency powers the security council will have and how they need to communicate these emergency actions to the broader community.
+
+## Abstract
+
+The snip answers the following questions:
+
+- What type of actions fall under “emergency action”?
+- When should such actions be used?
+- What is the legal quorum to make such a decision?
+- Is there any mandatory delay in the action?
+- How and when (before action ? after action?) to communicate this to the network?
+- Does anyone have veto powers over their decision ?
+
+## Motivation
+
+We want to not lose user funds.
```

Comment:
I think the motivation deserves to be a bit more detailed. Two sentences might suffice:

```suggestion
The utmost priority for a rollup is to never lose user funds. The decentralization of rollup control (e.g. 7-day delay on network upgrade) might go hinder the speed of reaction needed in case of emergency. The Security Council therefore should have the power to act quickly to mitigate a critical failure.
```